src/librustc_asan/lib.rs

```diff
@@ -9,18 +9,10 @@
 // except according to those terms.
 
 #![sanitizer_runtime]
-#![feature(alloc_system)]
 #![feature(nll)]
 #![feature(sanitizer_runtime)]
 #![feature(staged_api)]
 #![no_std]
 #![unstable(feature = "sanitizer_runtime_lib",
             reason = "internal implementation detail of sanitizers",
             issue = "0")]
-
-extern crate alloc_system;
-
-use alloc_system::System;
-
-#[global_allocator]
-static ALLOC: System = System;
```

Comment:
Note that this and the other sanitizer crates no longer link to `System` as it's not an available type to use. While a regression I think this is minor enough to allow because the default allocator is now `System` (not jemalloc) which all the sanitizers are compatible that. That, plus the fact that the sanitizers are unstable